scalableminds/webknossos
# Added Feature Highlight Page

I added a feature highlight page for a summary of the best high-level features. I also added a pricing page to feature our hosting solutions. Both should only be active on the public wk.org instance

Other changes:
- Updated the spotlight header slight to include links for features and pricing
- Added `/features` & `/pricing` routes
- Added Features & Pricing to navbar. Should only be available on public wK.org instance 
- Refactored "credits" footers as reusable component
- Created a `pages` folder to contain the new pages. Perhaps this can be reused for other page-like components
- added a bunch of icons and screenshots

### URL of deployed dev instance (used for testing):
- https://featurehighlight.webknossos.xyz/features
- https://featurehighlight.webknossos.xyz/pricing 

### Steps to test:
- Check content on dev instance
- Check that it only shows on wk.org. This is based of the `api/features` / `application.conf`: 
```
allowOrganizationCreation: true,
enableFrontpage: true
```

### Issues:
- fixes #3839 

------
